packages/key-did-resolver/src/@types/index.d.ts

```diff
@@ -0,0 +1,2 @@
+declare module 'bs58';
+declare module 'uint8arrays'
```

Comment:
Paul showed me a better way to do declarations: https://github.com/ceramicnetwork/js-did/blob/master/src/declarations.d.ts

Reply:
oh nice! I'll do that 👍